soc/ti/simplelink/cc23x0/CMakeLists.txt

```diff
@@ -0,0 +1,31 @@
+# Copyright (c) 2024 Texas Instruments Incorporated
+# Copyright (c) 2024 BayLibre, SAS
+#
+# SPDX-License-Identifier: Apache-2.0
+
+zephyr_sources(soc.c)
+zephyr_sources(ccfg.c)
+zephyr_include_directories(.)
+
+zephyr_linker_sources_ifdef(CONFIG_HAS_TI_CCFG SECTIONS ccfg.ld)
+
+SET(crc_tool_search_paths
+	/usr
+	/usr/local
+	/opt
+	$ENV{HOME}
+	$ENV{HOME}/.local/bin
+	$ENV{HOME}/.local/opt
+	$ENV{HOME}/bin)
```

Comment:
this looks wrong

Reply:
The crc-tool will be dropped at next PR update to avoid CI failure. This is not a problem because TI IDE (CCS) automatically signs the binary at build, before flashing it.

This crc-tool will be re-introduced when `west flash` command support is added in Zephyr.